catkin_tools/jobs/catkin.py

```diff
@@ -504,7 +504,7 @@ def create_catkin_build_job(context, package, package_path, dependencies, force_
             [MAKE_EXEC, 'install'],
             cwd=build_space,
             logger_factory=CMakeMakeIOBufferProtocol.factory,
-            locked_resource='installspace'
+            locked_resource=None if context.isolate_install else 'installspace'
```

Comment:
No more need to lock the install space if each package has its own one to write to.